File: OCH_Tao/ABC/B/177.py
S = input()
T = input()
ans = len(T)
for i in range(len(S)-len(T)+1):
  X = S[i:i+len(T)]
  cnt = 0
  for j in range(len(T)):
    if X[j]!=T[j]:
      cnt+=1
  else:
    ans = min(cnt,ans)
print(ans)

File: OCH_Tao/ABC/B/177.js
const [S, T] = require("fs").readFileSync("/dev/stdin", "utf8").trim().split("\n");
let ans = T.length;
for (let i = 0; i <= S.length - T.length; i++) {
  let cnt = 0;
  for (let j = 0; j < T.length; j++) {
    if (T[j] !== S[i + j]) {
      cnt++;
    }
  }
  ans = Math.min(ans, cnt);
}
console.log(ans);